uh_internal/apps/dailyduties/pyexchange.py

```diff
@@ -0,0 +1,38 @@
+import os
+from os.path import join, dirname
+from ...settings.base import get_env_variable
+
+# https://github.com/fedorareis/pyexchange This is a combination of a few branches and some custom code
+from pyexchange import Exchange2010Service, ExchangeNTLMAuthConnection, ExchangeBasicAuthConnection
+
+URL = get_env_variable('RESNET_INTERNAL_OUTLOOK_URL')
+USERNAME = get_env_variable('RESNET_INTERNAL_EMAIL_OUT_USERNAME')  # ResNet username
+PASSWORD = get_env_variable('RESNET_INTERNAL_EMAIL_OUT_PASSWORD')  # ResNet Password
+
+
+def setup():
+    """ Creates the Exchange Connection """
+    # Set up the connection to Exchange
+    connection = ExchangeBasicAuthConnection(url=URL,
+                                             username=USERNAME,
+                                             password=PASSWORD)
+
+    service = Exchange2010Service(connection)
+
+    return service
+
+
+def get_mail(service):
+    my_mail = service.mail()
+
+    list_emails = my_mail.list_mails()
+
+    return list_emails.count
+
+def get_voicemail(service):
+    
+    folder = service.folder()
+    giant_str = "AAMkADk3MzI3ZmNiLTM5YzMtNGZlOS1hZjVkLTFhN2I5ZTBjNmFmOAAuAAAAAACiNxZPdHhiS6q1zMiCAUIaAQBCzAjvjx3GTKwjaiEZoJadAAADOUuhAAA="
```

Comment:
Change this to a more meaningful string such as folder_id or voicemail_id or even better voicemail_folder_id